docs/cli/bunx.md

```diff
@@ -63,6 +63,8 @@ $ bunx --bun my-cli # good
 $ bunx my-cli --bun # bad
 ```
 
+Of course, you can also use the `#!/usr/bin/env bun` to execute scripts with bun.
```

Comment:
To force Bun to always be used with a script, use a shebang.

```ts
#!/usr/bin/env bun
```

Reply:
Think that may be a better way of saying this